services/client/src/styles/_variables.scss

```
@@ -0,0 +1,38 @@
+$color-bg-neutral: #f0f0f0;
+$color-bg-tag-auto: #e8f5e9;
+$color-bg-tag-user: #e0edf8;
+
+$color-border: #ddd;
+$color-border-light: #eee;
+$color-border-subtle: #e5e5e5;
+$color-border-input: #ccc;
+
+// ── Typography ──────────────────────────────────────────────
+
+$font-family: Arial, Helvetica, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)